pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -3066,6 +3067,7 @@ func handleIBCreate(ctx *domainContext, ib types.IoBundle) {
 			ib.Type, ib.Name, err)
 		return
 	}
+	// Preserve any existing Unique/MacAddr
```

Comment:
Same as above.. comment belongs in AddOrUpdateIoBundle ? Seems out of place here..

Reply:
It is actually stating an assumption in the caller. The fact that AddOrUpdate satisfies that assumption is a different matter. I can reword this with "We assume ..." if that makes sense.

Reply:
Done